.gitignore

```diff
@@ -54,4 +54,5 @@ test_for_zoneJS.ts
 
 migrations.*
 testDocs
-scully/tsconfig.sps-sample.json
\ No newline at end of file
+scully/tsconfig.sps-sample.json
+adsample.js
```

Comment:
I believe this line can be removed as this file is not present

Reply:
No, its something that lives on my computer, and I don't wnt it to get is ;)